src/vdaf/prg.rs

```diff
@@ -230,6 +246,35 @@ impl SeedStream for SeedStreamSha3 {
     }
 }
 
+#[cfg(feature = "experimental")]
```

Comment:
After looking closer at the sampler code in `dp/distributions.rs` I noticed that we're using `gen_range`. which implies `Rng`. Are there any other usages?

I think we pretty much already have an implementation of this, used for sampling pseudorandom field elements.

Reply:
the generator is used twice, once with `fill_bytes` (because its fast) and once with [`gen_range`](https://rust-random.github.io/rand/rand/trait.Rng.html#method.gen_range) (because it allows sampling uniform from a range of `BigUint` comfortably). the latter requires `Rng`. under various hoods it also uses `fill_bytes`, but it wouldn't be trivial and require duplicating some code from [`UniformBigUint`](https://docs.rs/num-bigint/latest/num_bigint/struct.UniformBigUint.html) to make it usable with a non-`Rng` rng...

you could always make an extra trait to enforce using a particular implementation in places where it's necessary. the `RngCore` trait basically is your `SeedStream` with some extra stuff (`try_fill` to error instead of panic,`next_u32`, `next_u64`) that's not very complicated to implement, but again not necessary for prio purposes.

Reply:
Filed #644.